replication/row_event.go

```diff
@@ -730,6 +740,22 @@ func (e *TableMapEvent) GeometryTypeMap() map[int]uint64 {
 	return ret
 }
 
+// VisibilityMap returns a map: column index -> visiblity.
+// Invisible column was introduced in MySQL 8.0.23
+// nil is returned if not available.
+func (e *TableMapEvent) VisibilityMap() map[int]bool {
```

Comment:
As for testing, I think maybe you can capture/construct some `VisibilityBitmap` bytes and test the result of TableMapEvent.VisibilityMap is expected.

Reply:
@lance6716 
Sure. I think I should create a separate function. Just as `TestTableMapOptMetaPrimaryKey` is defined solely for primary key type meta data. At first, I thought about squeezing into `TestTableMapOptMetaNames` or `TestTableMapHelperMaps`, but it's too packed and I saw no room for any addition. Your advice would be help me a lot. 😀